conformance/lang/values-types-and-variables/boolean-literal.balt

```diff
@@ -0,0 +1,161 @@
+Test-Case: output
+Description: Test boolean type descriptor.
+Labels: boolean-type-descriptor, boolean-literal
+
+public function main() {
+    boolean x = true;
+    io:println(x); // @output true
+
+    x = false;
+    io:println(x); // @output false
+}
+
+Test-Case: output
+Description: Test boolean literals with function calls.
+Labels: function-type-descriptor, boolean-type-descriptor, boolean-literal
+
+public function main() {
+    testBoolean(true);
+
+    testBoolean(false);
+
+    io:println(true.toString()); // @output true
+
+    io:println(false.toString()); // @output false
+}
+
+function testBoolean(boolean x) {
+    io:println(x); // @output true
+                   // @output false
+}
+
+Test-Case: output
+Description: Test boolean literals with return statement.
+Labels: function-type-descriptor, boolean-type-descriptor, return-type-descriptor, boolean-literal
+
+public function main() {
+    io:println(testBoolean(true)); // @output true
+
+    io:println(testBoolean(false)); // @output false
+}
+
+function testBoolean(boolean x) returns boolean {
+    return x;
```

Comment:
returning true and false should be part of the return statement IMO. We can write two functions with return statements like

```ballerina
    return true;
```

```ballerina
    return false;
```

Reply:
will add.